lib/ability.rb

```diff
@@ -20,11 +20,13 @@ def initialize(user, resource, controller = nil)
         can :manage, AccountPriceGroupMember
       else
         can :manage, :all
+        cannot([:edit, :update], User) { |target_user| !target_user.admin_editable? }
         unless user.billing_administrator?
           cannot [:manage_accounts, :manage_billing, :manage_users], Facility.cross_facility
         end
         unless user.account_manager?
           cannot :manage, User unless resource.is_a?(Facility) && resource.single_facility?
+          cannot([:edit, :update], User) { |target_user| !target_user.admin_editable? }
```

Comment:
Do we actually need this since we have it above? It's possible we do because of the `cannot :manage, User` in the previous line, but I just wanted to double check.